### PR TITLE
docs: fixing (most) Notion points

### PR DESCRIPTION
**Pull request by barjin**
This PR solves most of the minor problems with the Python Client docs mentioned in the [Notion page](https://www.notion.so/apify/Python-specific-Docs-V2-feedback-e3f0208fd1b94d6cb1a3e84d1e875156).

### CODE REVIEW

File: website/patches/docusaurus-plugin-typedoc-api+2.5.1.patch [hunk 1]
```diff
@@ -1,3 +1,27 @@
+diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberSignatureTitle.js b/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberSignatureTitle.js
```
Comment (fnesveda) [inline website/patches/docusaurus-plugin-typedoc-api+2.5.1.patch:1]: Holy shit what kind of magic are we doing here 😄 Wouldn't it be easier to fork the `docusaurus-plugin-typedoc-api` package and publish it under `@apfy-packages`?

Comment (B4nan) [reply website/patches/docusaurus-plugin-typedoc-api+2.5.1.patch:1]: patch-package is super simple solution, much simpler than forking and publishing to npm :]

Comment (fnesveda) [reply website/patches/docusaurus-plugin-typedoc-api+2.5.1.patch:1]: This would be a cool topic for an Eat&Learn or Tech Coffee 👍

Comment (B4nan) [reply website/patches/docusaurus-plugin-typedoc-api+2.5.1.patch:1]: I was actually mentioning that on tech coffee some time ago, together with `yarn link` :] 

You just modify the source inside `node_modules` and run `npx patch-package some-package`, the diff gets generated, and via `postinstall` hook it gets applied automatically (again just a CLI command). It gets tedious when you want to update the dependency, so not really a long-term solution, but super handy for debugging things in production, e.g. adding more logs to some package to test things in production.

Comment (fnesveda) [reply website/patches/docusaurus-plugin-typedoc-api+2.5.1.patch:1]: Ah, I see it in the history, I was just off that day I think 😄 Shame, looks cool!

**Pull request merged**